### PR TITLE
Support parent relationship

### PR DESCRIPTION
**Pull request by sql-dr-watson**

### CODE REVIEW

**Comment by sql-dr-watson** [comment]
@danhellem  let me know what you think and if you would like me to update the the changeling also.
thanks!

File: README.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ The id of the Work Item created or updated
    - `github_token`: Used to update the Issue with AB# syntax to link the work item to the issue. This will only work if the project is configured to use the [GitHub Azure Boards](https://github.com/marketplace/azure-boards) app. If you do not define this value, the action will still work, but the experience is not as nice.
    - `ado_bypassrules`: Used to bypass any rules on the form to ensure the work item gets created in Azure DevOps. However, some organizations getting bypassrules permissions for the token owner can go against policy. By default the bypassrules will be set to false. If you have rules on your form that prevent the work item to be created with just Title and Description, then you will need to set to true.
    - `log_level`: Used to set the logging verbosity to help with debugging in a production environment. 100 is the default. 
+   - `parentId`: Used to set the parent work item id for the work item created. This is useful when you want to create a child work item for a parent work item.
```
Comment (DariuszPorowski) [inline README.md:36]: to match existing naming convention, I think it should be a snake case.

**Comment by DariuszPorowski** [comment]
@danhellem @sql-dr-watson looking for this feature as well! :)

**Comment by DVAlexHiggs** [comment]
+1 Can we get this to completion? Was looking for a way to do this without having to reach out to the API manually.
Let me know if I can help

**Comment by danhellem** [comment]
Sorry, i was not paying attention to pending PR's like I should. I will take a look at this.

**Comment by danhellem** [comment]
I don't think I am going to merge this. It's not a common scenario and you can always fork the code and customize as you see fit. Closing :(

**Pull request closed without merging**